es6-destructuring/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script>
+      // javascript goes here
+      var book1 = {
+        title: 'Goodnight Punpun',
+        author: 'Inio Asano',
+        libraryID: 3353
+      };
+
+      const { title, author, libraryID: id } = book1;
```

Comment:
All three destructured variables should have the same names as the properties they pull from:

---

![image](https://user-images.githubusercontent.com/16143771/174711131-d83ff071-5f5f-4de4-bfd7-35e6c3d53a66.png)